m2m/m2m.py

```diff
@@ -185,62 +181,52 @@ def writeJSONFile(self, baseDirectory, foldername, data):
 
 if __name__ == '__main__':
 
-    usage = 'usage: %prog [options] <csv file>'
+    parser = ArgumentParser()
 
-    parser = OptionParser(usage=usage)
+    parser.add_argument('mapping',
+                        help='Specify the mapping file to use for this CSV')
 
-    parser.add_option('-m', '--mapping', action='store', type='string',
-                      dest='mapping',
-                      help='Specify the mapping file to use for this CSV')
+    parser.add_argument('file',
```

Comment:
I would change the name of this to perhaps `csv_file` to be less generic and because the current auto-generated usage could be more clear:
`usage: m2m.py [-h] [-n ROW] [-w] [-j] mapping file`